Jenkinsfile

```diff
@@ -54,6 +54,18 @@ def buildManifest = {String manifest, String bitbake_image ->
         sh "vagrant ssh -c \"/vagrant/vagrant-cookbook/yocto/build-images.sh ${yoctoDir} ${bitbake_image}\""
     }
 
+    stage("Copy images and cache ${bitbake_image}") {
+        sh "rm -rf archive"
+        sh "mkdir -p archive"
+        sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/tmp/deploy/images/ /vagrant/archive/\""
+        sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/downloads/ /vagrant/archive/\""
```

Comment:
For some reason docker seems to be the default provider, both on my computer and on the build server:

    Bringing machine 'default' up with 'docker' provider...